images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -56,6 +56,7 @@ $packageManagementList = @(
     (Get-HelmVersion),
     (Get-NPMVersion),
     (Get-NugetVersion),
+    (Get-ParcelVersion),
```

Comment:
- [Parcel is a zero configuration build tool for the web](https://github.com/parcel-bundler/parcel), not the package management utility
- We don't install parcel for windows-2022.

Reply:
ok

Reply:
Thanks. I fixed it.